docs/conf.py

```diff
@@ -105,6 +105,8 @@
 # If true, `todo` and `todoList` produce output, else they produce nothing.
 todo_include_todos = False
 
+# Extensions
+extensions = ['sphinx.ext.autosectionlabel']
```

Comment:
This is based on: https://sublime-and-sphinx-guide.readthedocs.io/en/latest/references.html#links-to-sections-in-the-same-document

Reply:
This is great!